synthtool/metadata.py

```diff
@@ -117,3 +118,14 @@ def remove_obsolete_files(old_metadata):
             os.unlink(file_path)
         except FileNotFoundError:
             pass  # Already deleted.  That's OK.
+
+
+def set_track_obsolete_files(track_obsolete_files=True):
+    """Instructs synthtool to track and remove obsolete files."""
+    global _track_obsolete_files
+    _track_obsolete_files = track_obsolete_files
+
+
+def should_track_obsolete_files():
+    global _track_obsolete_files
```

Comment:
I'm not a python expert, but do you need this if you're only accessing (not setting) the global, so there isn't any local/global ambiguity?

Reply:
It wasn't necessary.  Removed.